native/src/components/__tests__/NewsListItem.spec.tsx

```diff
@@ -35,7 +35,7 @@ describe('NewsListItem', () => {
   it('should correctly render a local news item', () => {
     const { getByText, queryByText } = renderNewsListItem(localNews, false)
     expect(getByText(localNews.title)).toBeTruthy()
-    expect(getByText('January 20, 2020')).toBeTruthy()
+    expect(getByText('20. Januar 2020')).toBeTruthy()
```

Comment:
For now the base language should still be english. Also in the tests. The language will be changed to system language in another PR.